python/src/aica_api/client.py

```diff
@@ -143,7 +148,7 @@ def controller_descriptions(self) -> requests.Response:
         """
         return requests.get(self._endpoint('controllers'))
 
-    def call_service(self, component: str, service: str, payload: str) -> requests.Response:
+    def call_component_service(self, component: str, service: str, payload: str) -> requests.Response:
```

Comment:
Do I need to deprecate this endpoint the same way as it is with `wait_for_predicate`?

Reply:
It would be better for user experience yes